docker-compose.yml

```diff
@@ -1488,6 +1482,18 @@ services:
         /arrow/ci/scripts/java_build.sh /arrow /build &&
         /arrow/ci/scripts/java_test.sh /arrow /build"
 
+  oracle-java:
```

Comment:
How is this different from `debian-java`? It seems to be based on the same base image.

Reply:
the maven image for openjdk 17 is based on Oracle Linux instead of debian `maven:3.8.5-openjdk-17`:
```
$ docker run --rm maven:3.8.5-openjdk-17 cat /etc/system-release
Oracle Linux Server release 8.5
```